.tests/integration/units.tsv

```diff
@@ -1,2 +1,2 @@
 sample	unit	run	lane	fastq1	fastq2	adapter
-NA12878	N	1	1	input/R1.fq.gz	input/R2.fq.gz	ACGT
+NA12878	N	1	1	prealigned/merged/fastq1.fq.gz	prealigned/merged/fastq2.fq.gz	ACGT
```

Comment:
Do you use `prealigned/merged/fastq1.fq.gz` and `prealigned/merged/fastq1.fq.gz` somewhere or why did you change it?

Reply:
Yes I use prealigned/merged/fastq1.fq.gz as this is how the output from the prealign module looks like.

Reply:
True, but you are not using these information from the `units.tsv`, right? I would not use these paths here because it creates confusion.

Reply:
Changed back.